packages/common/src/scopeSupportFacets/scopeSupportFacetInfos.ts

```diff
@@ -160,11 +160,20 @@ export const scopeSupportFacetInfos: Record<
     scopeType: "string",
   },
 
+  "branch.match": {
+    description: "A pattern match branch",
+    scopeType: "branch",
+  },
```

Comment:
this introduces a bit of confusion in Python, where a match statement is basically a switch statement with pattern matching